src/backend/main.ts

```diff
@@ -168,6 +169,9 @@ async function initializeWindow(): Promise<BrowserWindow> {
     Winetricks.download()
     if (!availableWine.length) {
       downloadDefaultWine()
+      if (isMac) {
```

Comment:
I don't know if this is the right place to check this though, cause the first time a user opens heroic, it will download a wine and run this check, but the second time it won't run this cause `availableWine.length` will be more than 0

I think this should be done somewhere else so we can show it in the logs always if needed